src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -204,7 +204,6 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<MonoSymbolArchive Condition=" '$(MonoSymbolArchive)' == '' And '$(EmbedAssembliesIntoApk)' == 'True' And '$(DebugSymbols)' == 'True' And '$(Optimize)' == 'True'" >True</MonoSymbolArchive>
 	<MonoSymbolArchive Condition=" '$(MonoSymbolArchive)' == '' ">False</MonoSymbolArchive>
 
-	<BundleAssemblies Condition="'$(BundleAssemblies)' == ''">False</BundleAssemblies>
```

Comment:
We might want to leave this one in place for the time being with a non conditional default value of False until we have a monodroid PR in place.
`<BundleAssemblies>False</BundleAssemblies>`

The reason being that MSBuild will error when trying to convert an empty string to a bool. 
Although if all the tests are passing then perhaps this isn't an issue.

Reply:
I will restore this line as-is, then. If we force it to `false`, then the warning added elsewhere for this won't work.